packages/ui/src/components/va-date-input/VaDateInput.vue

```diff
@@ -133,7 +133,7 @@ import { VaIcon } from '../va-icon'
 const VaInputWrapperProps = extractComponentProps(VaInputWrapper, ['focused', 'maxLength', 'counterValue'])
```

Comment:
<img width="461" alt="image" src="https://github.com/epicmaxco/vuestic-ui/assets/5394573/888b6d78-1bf8-4558-a147-7655cd8369ca">

Date input doesn't close on click outside.

Reply:
Same issue for time input

<img width="385" alt="image" src="https://github.com/epicmaxco/vuestic-ui/assets/5394573/e2c8879e-62b0-4f50-8e7f-20ab1b94faab">